src/app/features/visitor-form/visitor-form.component.scss

```diff
@@ -0,0 +1,32 @@
+@use '@angular/material' as mat;
+@import '../../styles/palettes';
```

Comment:
Powinniśmy używać `@use` zamiast `@import`, Grzesiu na discordzie opisał jak to robić